test/bundling.js

```diff
@@ -37,9 +37,6 @@ var createApiCall = require('./utils').createApiCall;
 var _ = require('lodash');
 
 function createSimple(value, otherValue) {
```

Comment:
Can we remove this function? 
Replacing `createSimple('value')` calls with `{field1: 'value', field2: undefined}` is more readable in tests and adds very little characters.